### PR TITLE
Add Basic SKU for ExpressRoute.

### PR DESCRIPTION
**Pull request by raraveen**
## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by cormacpayne** [comment]
@MikhailTryakhov should this be targeted at the `november-network-release` branch?

File: src/ResourceManager/Network/Commands.Network/ApplicationGateway/RedirectConfiguration/AzureApplicationGatewayRedirectConfigurationBase.cs [hunk 1]
```diff
@@ -78,20 +78,6 @@ public PSApplicationGatewayRedirectConfiguration NewObject()
             redirectConfiguration.Name = this.Name;
             redirectConfiguration.RedirectType = this.RedirectType;
 
-            if (this.TargetUrl != null
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/ApplicationGateway/RedirectConfiguration/AzureApplicationGatewayRedirectConfigurationBase.cs]: @raraveen any reason why these checks are being removed?

**Comment by MikhailTryakhov** [comment]
@cormacpayne contributor has problems with local branches so let's merge it any place we can :) 
I'll merge it to the monthly one after this PR would be ready

**Comment by cormacpayne** [comment]
@raraveen ping on the above comments

File: src/ResourceManager/Network/Commands.Network/ApplicationGateway/RedirectConfiguration/AzureApplicationGatewayRedirectConfigurationBase.cs (hunk 1 above)
Comment (raraveen) [reply src/ResourceManager/Network/Commands.Network/ApplicationGateway/RedirectConfiguration/AzureApplicationGatewayRedirectConfigurationBase.cs]: @cormacpayne - I have not made these changes. The ones I have made are at https://github.com/Azure/azure-powershell/pull/7833/commits/3ad9d2ea48eef3ea90bbb0f58df630853facd096

**Comment by cormacpayne** [comment]
@raraveen it looks like your first commit, https://github.com/Azure/azure-powershell/pull/7833/commits/3b908342c269eccfee9442419fd8670528490db9, introduced the extra changes that you're seeing in your PR. You can either [rebase](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/cleaning-up-commits.md#rebasing) to clean up the changes, or feel free to open a new PR in a new branch to introduce only your changes if rebasing is unsuccessful.

**Pull request closed without merging**